components/snackbar.css

```diff
@@ -0,0 +1,89 @@
+/* ---------------------------------Snackbar skelton--------------------- */
+.snackbar{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem;
+    width: max-content;
+    line-height: 1.5rem;
+    border-radius: 6px;
+    box-shadow: 4px 8px 1rem #747679;
+}
```

Comment:
Suggestion to use variable names for box-shadow as well.
Since this a component and the colors will change according to the website you implement it on.

Reply:
Fixed it.